controllers/pod_controller.go

```diff
@@ -46,10 +48,15 @@ type PodReconciler struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, err error) {
-	if !r.IgnoreNamespaces.In(req.Namespace) {
-		log.FromContext(ctx).V(1).Info("Triggering reconciliation")
-		r.Triggerer.Trigger()
+	for _, regex := range r.IgnoreNamespaces {
+		if regex.FindString(req.Namespace) != "" {
```

Comment:
I think what @MarcosDY was saying is that this call here can use the MatchString method on the compiled regex, which does not return an error. FindString is also fine just less intuitive since the result is throwaway.
```suggestion
		if regex.MatchString(req.Namespace) {
```